tests/bson/bson-timestamp_error-004.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+BSON BSON\Timestamp constructor requires positive 64-bit integers <= UINT32_MAX
```

Comment:
why does the title say positive 64-bit integers <= UNIT32_MAX? Surely, you could just say:

    requires positive unsigned 32-bit integers